### PR TITLE
Added pastor Controls

### PR DESCRIPTION
**Pull request by matt3501**
Hoping this syntax looks correct, I'm hoping to have a standard location for Old testament to be from the pulpit (exception Warnecke) and standard location for the sermon text (exception Krause)

### CODE REVIEW

File: order/bas/readme.md [hunk 1]
```diff
@@ -21,22 +21,24 @@ click 'Run Sub/User Form" and all break points should land on "Test Passed"
 
 Slide listener will listen for these tags and behave approprately.
```
Comment (paulbono) [inline order/bas/readme.md:23]: Why did you remove bulleted list?

**Review by paulbono — APPROVED** [review]

File: order/bas/readme.md (hunk 1 above)
Comment (matt3501) [reply order/bas/readme.md:23]: To better support a copy/paste workflow. I was afraid the bullet styling in the notes field when I would triple click--> copy-->paste would potentially affect the tag or it's detection (which it shouldn't, so really it was just because I was superstitious)
